README.md

```diff
@@ -1,199 +1,179 @@
 [![Build Status](https://travis-ci.org/igbt6/Packt-Publishing-Free-Learning.svg?branch=master)](https://travis-ci.org/igbt6/Packt-Publishing-Free-Learning)
 
-## Free Learning PacktPublishing script
+## Free Learning Packt Publishing script
 
-**packtPublishingFreeEbook.py** - script that automatically grabs and download a daily free eBook from https://www.packtpub.com/packt/offers/free-learning
-  You can use it also to download the already claimed eBooks from your account https://www.packtpub.com/account/my-ebooks
+`packt-cli` is a Python script that allows to automatically grab and download a daily Free
+Learning Packt ebook from https://www.packtpub.com/packt/offers/free-learning.
+You can also use it to download already claimed ebooks from your Packt
+account.
 
-The script uses [anti-captcha.com](https://anti-captcha.com/) service to bypass the Recaptcha captcha to function fully automatically. Anticaptcha.com employs people to solve captcha test use whether or not the user is human. The service costs about $ 2 per thousand captcha test, allowing you to operate for a few dollars over the years.
+The script uses [anti-captcha.com](https://anti-captcha.com/) service to bypass
+the Recaptcha captcha to function fully automatically. Anti Captcha employs
+people to solve captcha tests. The service costs about $2 per thousand captcha
+test, allowing you to operate for a few dollars over the years.
 
-### Requirements:
-* Install either Python 2.7 or 3.x
-* Install pip (if you have not installed it yet).
-  To install pip, download:  https://bootstrap.pypa.io/get-pip.py ,
-  then run the following command:
+### Installation
 
-  ```  
-  python get-pip.py
-  ```
-* Optionally install [*virtualenv*](http://docs.python-guide.org/en/latest/dev/virtualenvs/) (pip install virtualenv)
+To install script simply run
+```
+pip install git+https://github.com/igbt6/Packt-Publishing-Free-Learning.git@master

```

Comment:
Actually for testing this feature before merging you should use `pip install git+https://github.com/mjenczmyk/Packt-Publishing-Free-Learning.git@setup`